shared/src/main/scala/com/nitro/nmesos/sidecar/SidecarUtils.scala

```diff
@@ -49,25 +48,34 @@ object SidecarUtils {
         val sidecarInfo = sidecarEntries.filter(_.Status == 0)
           .map { s => s"${s.Image} @ ${s.Hostname}" }.sorted
 
-        val diff = sidecarInfo.diff(containerInfo)
-        if (diff.isEmpty) {
-          log.println(s""" ${log.Ok} Sidecar mapping for $serviceName match all containers running """)
-          sidecarInfo.foreach { info =>
-            log.info(s"\t\t$info")
-          }
-        } else {
-          log.println(s""" ${log.Fail} Invalid Sidecar mapping for $serviceName""")
-          log.error("\tExpected (Containers running):")
-          containerInfo.foreach { info =>
-            log.error(s"\t\t$info")
-          }
-          log.error("\tFound (in Sidecar):")
-          sidecarInfo.foreach { info =>
-            log.error(s"\t\t$info")
-          }
-        }
-        diff.isEmpty
+        diffInfo(containerInfo, sidecarInfo, serviceName)
     }.reduce(_ && _)
   }
 
+  def diffInfo(containerInfo: Seq[String], sidecarInfo: Seq[String], serviceName: String)(implicit log: Logger): Boolean = {
+    if (containerInfo.sameElements(sidecarInfo)) {
+      log.println(s""" ${log.Ok} Sidecar mapping for $serviceName match all containers running """)
+      sidecarInfo.foreach { info =>
+        log.info(s"\t\t$info")
+      }
+      true
+    } else if (containerInfo.isEmpty) {
+      log.info("\tFound (in Sidecar, but not in mesos):")
```

Comment:
```suggestion
       log.println(s""" ${log.Ok} Sidecar mapping for $serviceName found but not found on mesos hosts""")
```

Can you take `log.println` as the option here as well, to match the output from the other statements? :)